source/devices/AM62PX/linux/Linux_Performance_Guide.rst

```diff
@@ -1088,26 +1088,26 @@ Table:  **Deep sleep**
 .. csv-table::
     :header: "Rail name","Rail voltage(V)","am62pxx_sk-fs"
```

Comment:
please fix units of "am62pxx_sk-fs" - is that Watts? Watt hour? milliWatts? uWatts?

Reply:
fixed to be uW for AM62x, p, and a